barretenberg/cpp/src/barretenberg/goblin/goblin.hpp

```diff
@@ -88,6 +89,7 @@ class Goblin {
     AccumulationOutput accumulator; // Used only for ACIR methods for now
 
   public:
+    Goblin() { GoblinMockCircuits::perform_op_queue_interactions_for_mock_first_circuit(op_queue); }
```

Comment:
To mock inside Goblin I had to break the dependency between MockCircuits and Goblin (to avoid a circular dependency)